entrypoint.sh

```diff
@@ -203,8 +203,56 @@ function start_postfix {
         echo "OpenDKIM: Add TXT record to DNS:"
         cat /var/db/dkim/default.txt
     fi
+
     
     sed -n -e '/^Domain\s/!p' -e '$aDomain '$HOSTNAME -i /etc/opendkim/opendkim.conf
+    # DKIM for all virtual domains and $HOSTNAME 
+    if [ "$SMF_DKIM_ALL" != "" ]; then
+        if [ ! -f /var/db/dkim/$HOSTNAME/default.private ]; then
+            echo "Moving ${HOSTNAME} keys to /var/db/dkim/$HOSTNAME/"
+            mkdir -p /var/db/dkim/$HOSTNAME
+            cp /var/db/dkim/default.* /var/db/dkim/$HOSTNAME
+        fi
+            chmod 400 /var/db/dkim/default.private
+            chown opendkim:opendkim /var/db/dkim/default.private
+            echo "Inserting ${HOSTNAME} data to /etc/opendkim/{KeyTable, SigningTable, TrustedHosts}"
+            echo "default._domainkey.${HOSTNAME} ${HOSTNAME}:default:/var/db/dkim/${HOSTNAME}/default.private
+            " >> /etc/opendkim/KeyTable
+
+            echo "${HOSTNAME} default._domainkey.${HOSTNAME}" >> /etc/opendkim/SigningTable
+
+            echo "${HOSTNAME}" >> /etc/opendkim/TrustedHosts
```

Comment:
Please fix the intention. And if you append with `>>`, does it mean that on every SMF restart new lines will be added to  `/etc/opendkim/{KeyTable, SigningTable, TrustedHosts}`?

Reply:
Thanks for checking! Indentation fixed at line 220, unless you meant something else.

The `>>` is required, because for all files I am inserting infos on multiple occasions:
-  first for the entry your PR creates (the ${HOSTNAME})
- over a loop for all `virtualDomain in $virtualDomains`

All new files in `/etc/opendkim/` are re-created at each `restart` or `down/up`, they are dynamic according to the inserted domains, so there is no danger of inserting new lines.
```
docker exec -it smf cat /etc/opendkim/KeyTable | wc -l
10
docker-compose restart
docker exec -it smf cat /etc/opendkim/KeyTable | wc -l
10
```

Reply:
This was a good fix, same thing on line 234.

But I meant that the whole block of code from lines 216-223 and 231-242 is intented:
![image](https://user-images.githubusercontent.com/2674369/107503519-baee8580-6ba2-11eb-84e4-0253def7e6ea.png)

Reply:
Whops! Thanks for pointing it out. Fixed.